### PR TITLE
Survey fix

### PR DESCRIPTION
**Pull request by nastasi-oq**
Rewrited csv to json converter to fix a bug in the case a column must be completely skipped.
All the stuff needed to convert source dataset file now is included in the repository (data folder).
Tests are running here: https://ci.openquake.org/job/zdevel_oq-platform/427/

### CODE REVIEW

**Review by lmartins88 — APPROVED** [review]

**Review by lmartins88 — APPROVED** [review]

**Review by lmartins88 — APPROVED** [review]

**Comment by daniviga** [comment]
Please add an `hotfix` tag as we do with Platform, so we know that code should be pushed in production as soon as PR is merged (instead of waiting for the next Platform scheduled release).

**Review by daniviga — APPROVED** [review]
LGTM

**Pull request merged**
